test/query_test.rb

```diff
@@ -51,40 +51,36 @@ def test_quote
     assert_query quoted, Fauna::Query.quote(quoted)
   end
 
-  def test_lambda
-    assert_equal Fauna::Query.lambda { |a| Fauna::Query.add(a, a) }, {
+  def test_lambda # rubocop:disable Metrics/MethodLength
+    assert_equal Fauna::Query.lambda { |a| Fauna::Query.add(a, a) },
       lambda: 'auto0',
       expr: { add: [{ var: 'auto0' }, { var: 'auto0' }] }
-    }
 
     lambda = Fauna::Query.lambda do |a|
       Fauna::Query.lambda do |b|
         Fauna::Query.lambda { |c| [a, b, c] }
       end
     end
-    assert_equal lambda, {
```

Comment:
Rubocop warns if the last parameter is a hash and you put in unnecessary curly braces.
For hashes with curly braces, it wants you to put a comma on the last line.